charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml

```
@@ -9,39 +9,54 @@ metadata:
     app: {{ .Values.adotCollector.daemonSet.configMap.app }}
     component: {{ .Values.adotCollector.daemonSet.configMap.component }}
 data:
-  adot-config: |
+  adot-config:  |
     extensions:
       health_check: {{ .Values.adotCollector.daemonSet.extensions.healthCheck }}
     receivers:
       awscontainerinsightreceiver:
-        collection_interval: {{ .Values.adotCollector.daemonSet.receivers.collectionInterval }}
-        container_orchestrator: {{ .Values.adotCollector.daemonSet.receivers.containerOrchestrator }}
-        add_service_as_attribute: {{ .Values.adotCollector.daemonSet.receivers.addServiceAsAttribute }}
-        prefer_full_pod_name: {{ .Values.adotCollector.daemonSet.receivers.preferFullPodName }}
+        collection_interval: {{ .Values.adotCollector.daemonSet.cwreceivers.collectionInterval }}
+        container_orchestrator: {{ .Values.adotCollector.daemonSet.cwreceivers.containerOrchestrator }}
+        add_service_as_attribute: {{ .Values.adotCollector.daemonSet.cwreceivers.addServiceAsAttribute }}
+        prefer_full_pod_name: {{ .Values.adotCollector.daemonSet.cwreceivers.preferFullPodName }}
+      prometheus:
+        config:
+          global:
+            scrape_interval: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeInterval }}
+            scrape_timeout: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeTimeout }}
+          scrape_configs: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeConfigs | nindent 10 }}
```

```suggestion
          scrape_configs: 
            {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeConfigs | nindent 12 }}
```

Pretty sure this needs to start on the next line and be indented 12 spaces.